client/state/notices/middleware.js

```diff
@@ -76,6 +78,17 @@ export function onPostSaveSuccess( dispatch, action ) {
 	}
 }
 
+export function onGuidedTransferHostDetailsSaveFailure( dispatch, { error } ) {
+	if ( error && error.message ) {
+		dispatch( errorNotice( error.message ) );
+	} else {
+		dispatch( errorNotice( translate( 'Whoops, there was an error saving your details. Please ' +
```

Comment:
Previously the error messages were being read from the server, so I couldn't include links in the text. In 25feeb4 I've added an inline notice which uses the error code and adds a Get Help link:
![screen shot 2016-09-02 at 12 10 15 pm](https://cloud.githubusercontent.com/assets/416133/18190421/41d2a8b6-7106-11e6-88f7-56416c1afa0d.png)

Or, alternatively, how about the error on top?
![screen shot 2016-09-02 at 11 58 58 am](https://cloud.githubusercontent.com/assets/416133/18190490/9b6a29bc-7106-11e6-8e30-1029a66fcc2f.png)